### PR TITLE
Fixed details on fallback behaviour to find sender email address in workflows.

### PR DESCRIPTION
**Pull request by AndyButland**
This came up in a support issue from where I realised we were missing some detail of the fallback behaviour here.  Can be reviewed and published at any time.

### CODE REVIEW

File: 10/umbraco-forms/editor/attaching-workflows/workflow-types.md [hunk 1]
```diff
@@ -87,22 +87,41 @@ Sends the result of the Form to the specified email address. The following confi
...
-                    }
                 }
             }
+        }
+    }
 ```
 
-* Reply To Email
-* Subject of the email (required)
+The fallback behavior to find the sender email address described also applies to the other email workflows.
```
Comment (sofietoft) [inline 10/umbraco-forms/editor/attaching-workflows/workflow-types.md:124]:
```suggestion
The fallback behavior also applies to the other email workflows.
```

File: 11/umbraco-forms/editor/attaching-workflows/workflow-types.md [hunk 2]
```diff
@@ -83,26 +83,45 @@ Sends the result of the Form to the specified email address. The following confi
...
-                    }
                 }
             }
+        }
+    }
 ```
 
-* Reply To Email
-* Subject of the email (required)
+The fallback behavior to find the sender email address described also applies to the other email workflows.
```
Comment (sofietoft) [inline 11/umbraco-forms/editor/attaching-workflows/workflow-types.md:120]:
```suggestion
The fallback behavior also applies to the other email workflows.
```

**Review by sofietoft — COMMENTED** [review]
Looks good @AndyButland !
I took the liberty to fix some links.
Also added a couple of comments.

**Comment by AndyButland** [comment]
Thanks, I've committed the suggestions.

**Review by sofietoft — APPROVED** [review]

**Pull request merged**